RasimKhusaenov/my_own_blog
# Add ability to create an article by company users

### Summary

- Refactored companies (it uses subdomains now)
- Added ability to create an article by company users

### How it works

![image](https://user-images.githubusercontent.com/35406008/184527064-4fcb8900-2941-4b71-9852-69b7f1451ec3.png)
![image](https://user-images.githubusercontent.com/35406008/184527119-994d8b36-a42d-40c1-aff2-1efcd181fdb9.png)
![image](https://user-images.githubusercontent.com/35406008/184527249-2477ed6b-cd32-486a-9410-075d07cf8a8d.png)

### Test plan

List of steps to manually test introduced functionality:

- Log in as company writer (login: reader@example.com; password: 123456)
- Go to http://flatstack.khusaenov.ru/users/articles/new
- Fill form
- Click "Create Article"
- Make sure article has been created

### Deploy notes

Run
```ruby
rails db:migrate
```

### Checklist:


### References
- Resolves #39 Issue
